core/src/main/scripts/importer/validateData.py

```diff
@@ -4253,6 +4253,7 @@ def processCaseListDirectory(caseListDir, cancerStudyId, logger,
                                 'all_cases_with_mutation_and_cna_data',
                                 'all_cases_with_mutation_and_cna_and_mrna_data',
                                 'all_cases_with_gsva_data',
+                                'all_cases_with_structural_variant_data',
```

Comment:
Same here, should be `all_cases_with_sv_data`

Reply:
okay, will make corrections. thanks @rmadupuri!

Reply:
done.